### ISSUE TITLE
Install only enabled hooks

### ISSUE THREAD

**Issue by prudloff-insite** [captainhookphp/hook-installer#2]
Currently this plugin installs every hook even if they are disabled. This can cause a lot of `hook is disabled` messages when triggering disabled hooks.
It would be nice to have an option to call `captainhook install --only-enabled` (option added in https://github.com/captainhookphp/captainhook/issues/174).

**Comment by sebastianfeldmann** [comment]
I just added a config option to trigger the `--only-enabled` option for the install command.

```
{
  "extra": {
    "captainhook": {
      "exec": "tools/captainhook.phar",
      "force-install": true,
      "only-enabled": true
    }    
  }  
}
```

Hope that helps :)

Fixed in version `1.0.2`

**Comment by sebastianfeldmann** [comment]
I totally forgot. You can now run `captainhook uninstall --only-disabled` as well.

This should, at least, make it a bit easier if you installed all hooks by accident.
